packages/react/src/components/table/components/SortingHeaderCell/SortingHeaderCell.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+// import core base styles
+import 'hds-core';
+import styles from '../../Table.module.scss';
+import {
+  IconSort,
+  IconSortAlphabeticalAscending,
+  IconSortAlphabeticalDescending,
+  IconSortAscending,
+  IconSortDescending,
+} from '../../../../icons';
+
+export type SortingHeaderCellProps = React.ComponentPropsWithoutRef<'th'> & {
+  ariaLabelSortButtonNeutral: string;
+  ariaLabelSortButtonAscending: string;
+  ariaLabelSortButtonDescending: string;
+  colKey: string;
+  setSortingAndOrder: Function;
+  order: 'neutral' | 'asc' | 'desc';
```

Comment:
Another name for neutral could be 'unset'? Neutral is ok too

Reply:
Unset feels better, let's change it.

- [ ] Change neutral to unset